.travis.yml

```diff
@@ -60,7 +60,7 @@ before_script:
 
 # command to run tests
 script:
- - nosetests -vs --with-coverage --cover-package=pysat
+ - nosetests -vs --with-coverage --cover-package=pysat.tests
```

Comment:
Why is this changed?

Reply:
It didn't work otherwise. I was doing most of the testing with `nosetests pysat.tests.whatever` so I didn't catch it at first. I added an import for tests in init and that didn't do the trick either. Something related to the wrapper class I presume.